### DISCUSSION TITLE
structure of fparam

### DISCUSSION THREAD

**Discussion by chtchelkatchev** [deepmodeling/deepmd-kit#4282] [Q&A] [answered]
From the discussion https://github.com/deepmodeling/deepmd-kit/discussions/3466 follows that fparam for a frame can be not a scalar but two or more scalers as well. So, is it true, that fparam for a given frame can be equal to [scaler1, scaler2,..., scalerN], where N is a fixed number of frame parameters?

**Comment by njzjz** [comment, marked as answer]
You need to set `numb_fparam` to $N$.

**Reply by chtchelkatchev** [reply]
Thank You for quick answer!
